common/gnome-shell/3.18/sass/_common.scss

```diff
@@ -125,7 +125,7 @@ StScrollBar {
   StButton#vhandle, StButton#hhandle {
     border-radius: 4px;
     background-color: mix($fg_color, $bg_color, 40%);
-    border: 0px solid;
```

Comment:
Technically this is not a typo, or in any way incorrect, although defining the `solid` border-style seems completely redundant for 0px wide border. At least the commit message could be changed to reflect that.

I'm wondering though, whether the border-style could have been defined on purpose due to some gnome-shell CSS oddity. Just wondering if you have investigated this any furter? If you tested that the scrollbar handles look as they should after this, I suppose this should be ok for merging.

Reply:
You're correct that this is not technically a typo, the redundant border style was just bugging me - I will amend the commit message to be more accurate.

I have tested this visually, and scrollbar handles are unchanged with this edit.

The upstream code actually doesn't have a border property for the scrollbar handles (there is a commented out border property), so this was added by horst for the Arc theme.

Reply:
I think it should be ok then, to remove the border definition altogether, if that doesn't change the appearance.

I'm not sure I understand the purpose of the commented out border definition in upstream, as the explanation, `//would be nice to margin or at least to transparent` makes little sense to me. My best guess is that the 3px margin could be replaced with 3px transparent border, which would allow the scrollbar handles to be click-and-dragged from the margin area.

If upstream adds the border at some point, it can be then updated for Arc, but for now the border definition seems completely redundant to me.

Reply:
After some testing, removing the `border` property completely does not alter the visual appearance of the scrollbar handles; have tested both with and without the Arc theme compiled into the `gresource` file.

So I do agree that the best option is to remove the `border` property completely, seeing as it is absent in upstream.